### DISCUSSION TITLE
Promotional banner

### DISCUSSION THREAD

**Discussion by rmccar** [ONSdigital/design-system#730] [Feedback]
Use this place to discuss the [promotional banner](https://ons-design-system.netlify.com/promo-banner) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Default promotional banner graphic overlapping `promo-banner__content` preventing links being clickable

- Place a link in the lower portion of `promo-banner__content`
- Graphic prevents click

Adding the following css to `promo-banner__content` fixes this issue...
```
z-index: 1;
position: relative;
```

**Reply by jrbarnes9** [reply]
#1292 resolved

**Comment by jrbarnes9** [comment]
Promo banner should use wider grid col if no image

**Reply by jrbarnes9** [reply]
#1426 resolved

**Comment by jrbarnes9** [comment]
Promo banner should allow setting of background image graphic and use white text by default

**Reply by jrbarnes9** [reply]
Raised #2167
